scripts/local_facts.fact

```diff
@@ -36,7 +35,9 @@ if [ -f /etc/rpi-issue ]; then
     OS="raspbian"
 fi
 
-if tmp=$(grep ^VERSION_ID= /etc/*elease); then
+# /etc/lsb-release could also be grep'd.  But /etc/upstream-release/lsb-release
+# on Linux Mint 20 caused grep of /etc/*elease to fail (on directory not file)
+if tmp=$(grep ^VERSION_ID= /etc/os-release); then
```

Comment:
Any reason not to have line 30 the same as the change in line 40? It's intended to be the same file that is referenced.

Reply:
Thanks for catching this.  It's serious and should be fixed by PR #2596